client/components/sideCarousel.jsx

```diff
@@ -19,18 +20,41 @@ const Image = styled.img`
   height: 60px;
   width: 60px;
   border-radius: 6px;
+  opacity: 0.6;
+`;
+
+const ClickedListItem = styled.li`
```

Comment:
I don't know anything about styling yet - just double checking that these are supposed to start with a capital letter. (I'm sure they are)